site/docs/v1/tech/apis/scim/SCIMServiceProvider.adoc

```diff
@@ -4,12 +4,94 @@ title: SCIM Service Provider API Overview
 description: APIs for discovering configuration information about FusionAuth SCIM Service Provider using a SCIM Client
 ---
 
-This API is used to retrieve information about the configuration of FusionAuth SCIM Service Provider per the SCIM specification.
+:unauthenticated:
+:no_missing:
```

Comment:
I was able to get a 404 status code for schemas, if not for the others.

curl -vvv -X GET https://sandbox.fusionauth.io/api/scim/resource/v2/Schemas/abc

Reply:
ah, good point. I was thinking of the root endpoint not getting a 404

Reply:
fixed